libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Cert.hs

```diff
@@ -25,8 +25,10 @@ certEnvSpec ::
   Specification fn (CertEnv (ConwayEra StandardCrypto))
 certEnvSpec =
   constrained $ \ce ->
-    match ce $ \_ pp _ _ _ ->
-      satisfies pp pparamsSpec
+    match ce $ \_slot pp _currEpoch _currCommittee proposals ->
+      [ satisfies pp pparamsSpec
+      , assert $ genHint 0 proposals
```

Comment:
Does this mean that proposals will always be empty?

Reply:
No it means that we prefer them to be empty, but there is no hard constraint forcing it.
Just a condition to make debugging easier in may cases, because there will be less noise.

Reply:
I don't really like the idea of constraining the sizes using constraints. For testing we should generate as large of an example as possible so we're more likely to catch bugs. We can always add size constraints later when reproducing a bug.

Reply:
OK I removed it.